### PR TITLE
Identification Console PR

### PR DESCRIPTION
**Pull request by GoldenAlpharex**
## About The Pull Request

It simply adds a new Modular Console Preset to the existing ones, fully outfitted to replace the current Identification Console in the Head of Personnel's Office and in the Bridge, and in most of the maps' Customs desks. Note that it doesn't replace them, it's just an asset right now so we can replace them with another PR, at the pace the map maintainers feel confident with. It has the ID Card Modification, Job Manager and Crew Manifest softwares on it by default, alongside the regular Chat Client.
The new Modular Console Preset can be found under **/obj/machinery/modular_computer/console/preset/id**, for all your map editing needs.
I'm planning to see if I can't make changes to the Departmental Consoles later on as well, so we basically can get rid of a console in every Head of Staff's office (except the HoP because there's only one), but that will require some additional coding to make it so it's not straight up some Identification Consoles that people with All-Access could use to print even more All-Access cards.

## Why It's Good For The Game

I've basically just implemented something that I feel was meant to be implemented a while ago through the use of TGUI and Modular Computers, but never actually got done. The console doesn't have the glitches of the current Identification console, when you remove your ID after logging in and it just goes to a page saying that the address couldn't be found. The UI is much simpler, looks a lot better and is just overall an improvement.

Here's what the new UI looks like, for those of you that don't feel like checking out the PR in-game to know what it looks like:
image
image

## Changelog
:cl:
add: New Modular Console Preset added: /obj/machinery/modular_computer/console/preset/id
/:cl:

### CODE REVIEW

**Comment by HomocidlePancakes** [comment]
But why?

**Comment by GoldenAlpharex** [comment]
> But why?

What do you mean, but why?

**Comment by HomocidlePancakes** [comment]
Edited the last comment.

**Comment by GoldenAlpharex** [comment]
> But why not just change the UI on the current console?

I'm no coding expert, I just did some very simple edits because it just felt right. Try it, honestly it's just fantastic. I can provide screenshots of what it looks like if you really want.

**Comment by HomocidlePancakes** [comment]
I know how modular console ID UI looks, I agree, it’s better.

**Comment by GoldenAlpharex** [comment]
> I know how modular console ID UI looks, I agree, it’s better.

Honestly, it all came from me not playing on Delta Station for a while and finally playing it again yesterday, it's the only map that has the Command Console and it's just so much better than what the HoP has. I know that it's already accessible by using the Hop's Tablet, but I thought that we probably just take the leap and just upgrade. I haven't removed the old console yet, this is just an asset so map makers can change it when they feel ready for it.

**Comment by GoldenAlpharex** [comment]
Alright, I added some pictures to the description, so people can appreciate the nice Step up from what's currently available for HoPs.

**Comment by MrDoomBringer** [comment]
HOPs could always use this UI from their tablets but yeah its nice to have it in computer form now

File: code/modules/modular_computers/computers/machinery/console_presets.dm [hunk 1]
```diff
@@ -39,6 +39,7 @@
 	hard_drive.store_file(new/datum/computer_file/program/alarm_monitor())
 	hard_drive.store_file(new/datum/computer_file/program/supermatter_monitor())
 
+
```
Comment (MrDoomBringer) [inline code/modules/modular_computers/computers/machinery/console_presets.dm:42]:
```suggestion
```

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]
There's a pretty critical issue that needs addressing here

File: code/modules/modular_computers/computers/machinery/console_presets.dm (hunk 1 above)
Comment (GoldenAlpharex) [reply code/modules/modular_computers/computers/machinery/console_presets.dm:42]: Huh? What does that change, exactly? I'm still new to GitHub.

Comment (MrDoomBringer) [reply code/modules/modular_computers/computers/machinery/console_presets.dm:42]: lmao it doesn't actually affect how your code works. it's just convention to have consistent spacing between chunks of code for readability. This changes it from 2 blank lines to 1

Comment (GoldenAlpharex) [reply code/modules/modular_computers/computers/machinery/console_presets.dm:42]: Oh, okay, it's just that one of them had two, so I thought I'd make them all have two. My bad, will commit.

**Comment by ArcaneMusic** [comment]
A long time ago, the dream was to move all computer UIs as apps, then replace all computers with just modulars.
Neat change.

**Comment by GoldenAlpharex** [comment]
> A long time ago, the dream was to move all computer UIs as apps, then replace all computers with just modulars.
> Neat change.

I mean, that's honestly what I want to do, this was just my first PR. Modular computers are simply better, there's nothing else to say. Only thing stopping me from doing it right now is that I'd need to change the code for the IDs to be departmental when it's on a departmental console, or else it would just be Identification consoles in every department, making it super easy for someone that got AA to make a bunch more AA's.

**Comment by Ryll-Ryll** [comment]
Technically the standard access console can edit the registered age of an ID card, would you be kind enough to add that to this?

**Comment by GoldenAlpharex** [comment]
> Technically the standard access console can edit the registered age of an ID card, would you be kind enough to add that to this?

I mean, I can see what I can do about it, but I'm not the one who did the whole code behind it, this is merely just putting together existing assets. Although, to be completely fair with you, there's not a single time I've actually seen age matter in this game, although I guess there's the underage part for the bar, but I think that being underage shouldn't really a thing on SS13 as a whole, because honestly, why hire children when you can hire competent adults for exactly the same price? Anyway, that's more of a discussion for another day.

**Comment by Doosteroni** [comment]
Smells like an emagged laptop/tablet buff to me. Always good!

**Comment by GoldenAlpharex** [comment]
Alright, well, how do I ask Maintainers to merge this PR?

**Comment by optimumtact** [comment]
Thanks for the contribution and sorry for the delay

**Pull request merged**